components/browser/errorpages/src/main/res/values/strings.xml

```diff
@@ -185,12 +187,13 @@ be temporary, and you can try again later.</li>
 </ul>
      ]]></string>
 
-    <!-- NavigationDelegate.ERROR_CATEGORY_URI -->
+    <!-- Category: NavigationDelegate.ERROR_CATEGORY_URI -->
     <string name="error_unknown_host_title">Address Not Found</string>
+    <!-- LOCALIZATION NOTE (error_unknown_host_message) in the example, the two URLs in markup do not need to be translated. -->
     <string name="error_unknown_host_message"><![CDATA[
 <p>The browser could not find the host server for the provided address.</p>
 <ul>
-<li>Did you make a mistake when typing the domain? (e.g. <q><strong>ww</strong>.mozilla.org</q> instead of <q><strong>www</strong>.mozilla.org</q>)</li><li>Are you certain this domain address exists?  Its registration may have expired.</li>
+<li>Did you make a mistake when typing the domain? (e.g. <q><strong>ww</strong>.mozilla.org</q> instead of <q><strong>www</strong>.mozilla.org</q>)</li><li>Are you certain this domain address exists?  It\'s registration may have expired.</li>
```

Comment:
Should this be a proper English apostrophe?

Also, if this changes the string from their corresponding messages in Focus etc, maybe we should fix these once we've set up l10n etc? We'll like look into the English string to carry forward localizations so that we don't loose existing work, and this might make that trickier.

Reply:
I confirm this should not be a straight apostrophe. 
(' vs ’)

Reply:
I assumed it was an error, but I see your point as well. :)

Reply:
Thanks for confirming Delphine!